### PR TITLE
adds the ability to enter offline payments for manual invoices

### PR DESCRIPTION
**Pull request by cgerrior**
(enterprise feature). Tests included. https://docs.recurly.com/api/invoices#offline-manual-payment.

### CODE REVIEW

File: src/test/java/com/ning/billing/recurly/TestRecurlyClient.java [hunk 1]
```diff
@@ -587,6 +587,37 @@ public void testCreateAndCloseInvoices() throws Exception {
 
             final Invoice failedInvoice = recurlyClient.markInvoiceFailed(invoiceFail.getInvoiceNumber());
             Assert.assertEquals(failedInvoice.getState(), "failed", "Invoice not failed successfully");
+			
+            // Create an Adjustment
+            final Adjustment c = new Adjustment();
```
Comment (pierre) [inline src/test/java/com/ning/billing/recurly/TestRecurlyClient.java]: Could you put the enterprise-only checks in their own tests and group, so I can still run the `integration` ones before releases?

File: src/main/java/com/ning/billing/recurly/RecurlyClient.java [hunk 2]
```diff
@@ -558,6 +558,18 @@ public Invoice markInvoiceSuccessful(final Integer invoiceId) {
     public Invoice markInvoiceFailed(final Integer invoiceId) {
         return doPUT(Invoices.INVOICES_RESOURCE + "/" + invoiceId + "/mark_failed", null, Invoice.class);
     }
+	
+	/**
+	* Enter an offline payment for a manual invoice (beta)
```
Comment (pierre) [inline src/main/java/com/ning/billing/recurly/RecurlyClient.java]: Let's add a comment in the Javadoc that it's an enterprise-only feature.

File: src/test/java/com/ning/billing/recurly/TestRecurlyClient.java (hunk 1 above)
Comment (cgerrior) [inline src/test/java/com/ning/billing/recurly/TestRecurlyClient.java]: I think I've done it correctly. I'll make sure to add any enterprise features to the correct test in the future. Thanks!

Comment (pierre) [inline src/test/java/com/ning/billing/recurly/TestRecurlyClient.java]: Perfect! The only modification I've done was to remove the `alwaysRun = true` from the `@BeforeMethod` and `@AfterMethod` annotations, and list the groups explicitly (that way, they are not run when running the `fast` tests).

FYI, interestingly enough, the test `testCreateAndCloseManualInvoices` now passes on my test tenant.

**Pull request merged**